cogs/twitchlisten.py

```diff
@@ -16,22 +17,11 @@ def __init__(self, bot):
         self.is_live = False
         self.last_seen_live = False
 
-        # channels to send to
-        base_path = (Path(os.path.abspath(__file__)) /'..' /'..').resolve()
-        self.config_path = base_path / 'config_data.json'
-        with open(self.config_path) as json_file: # add path thing
-            config_data = json.load(json_file)
-            self.channels = config_data['twitch_announcement_channels']
-
-
-        
-        # twitch and discord channel info 
+        # twitch channel info
         load_dotenv()
         self.client = os.getenv("TWITCH_CLIENT")

```

Comment:
Oh and this needs to incorporate the changes from `main` if you'd be manually copying. Though, here it *would* have actually merged nicely if not for the other merge conflicts like the `TWITCH_DISCORD_CHANNEL` removal :cry: